pandas/tests/indexes/test_base.py

```diff
@@ -2517,6 +2581,10 @@ def test_generated_op_names(opname, indices):
         # pd.Index.__rsub__ does not exist; though the method does exist
         # for subclasses.  see GH#19723
         return
+    if (isinstance(index, RangeIndex) and
+        opname in ['add', 'radd', 'sub', 'rsub',
```

Comment:
why this?

Reply:
Same as above.  Adding `RangeIndex` to pandas/tests/indexes/conftest.py revealed that the implementation of the operators for `RangeIndex` is done differently.  Namely, for `RangeIndex`, the op methods are bound to one function with a parameter that includes the opname, but for the other index types, there are direct methods for each of the operators.

Reply:
Was a little tricky, but I think I figured it out right.